server/graphql/plugin/src/plugin_data/query.rs

```diff
@@ -12,8 +11,7 @@ use service::auth::{Resource, ResourceAccessRequest};
 
 #[derive(Union)]
 pub enum PluginDataResponse {
-    Response(PluginDataNode),
-    Error(NodeError),
+    Response(PluginDataConnector),
```

Comment:
Just making this a connector, since for column plugins we want to query for data once